homeassistant/components/person/__init__.py

```diff
@@ -446,6 +447,8 @@ def extra_state_attributes(self):
             data[ATTR_SOURCE] = self._source
         if (user_id := self._config.get(CONF_USER_ID)) is not None:
             data[ATTR_USER_ID] = user_id
+        if self.device_trackers is not None:
+            data[ATTR_DEVICE_TRACKERS] = self.device_trackers
```

Comment:
We should probably exclude this attribute from being recorded in the DB

example: https://github.com/home-assistant/core/blob/dev/homeassistant/components/update/recorder.py

Reply:
So, I've followed this example, but I'm really struggling with testing.  I've added tests for the exclusion in the recorder, but they're presently flaky for some reason (device_trackers does show up sometimes) and I'm not sure why.  Could use your help/advice if you have any.